### ISSUE TITLE
Re-sizing the VM does nothing - also does not reflect outside changes.

### ISSUE THREAD

**Issue by YouveGotMeowxy** [docker/for-win#5881]
I am using the latest Edge version, and referring to this:

![image](https://user-images.githubusercontent.com/39491297/75294503-328a9f80-57ee-11ea-98df-0d6467677daf.png)

I recently ran out of space (It was originally set at 60 gb) and needed to resize the VM.

I first tried simply sliding this bar over to 80 gb and clicking the "update & restart* button. All that changed was where the slider displays, as it didn't actually do any re-sizing of the actual VM. This was extremely confusing, as I kept getting the same exact errors I was when it was too small and out of space, while I thought the VM was a new size with plenty of new space ...

After more time spent (wasted :p) I realized that while the slider showed the new size, the VM wasn't any different, so I re-sized it to 100 GB using Window's Hyper-v edit VM wizard. This worked fine, however, upon restarting DD and looking at the slider agian, it was still show as 80 gb (not recognizing it was actually now 100 gb). 

Realizing this slider is probably just a "dummy" slider that does nothing, just for the heck of it I tried sliding it over to 100 gb just as a sort of reminder of it's size, but it wouldn't let me set it at an even 100, the next option was 104, so I just left it there for now.

It would be most excellent if:

a). It was able to actually reflect the real size of the VM.

b). Actually resize the VM when sliding it.

It seems as if since it's pretty much useless, it might just be better to show it's (real) size as a label instead of a slider, with a link to the Windows Wizard to Extend the size.

**Comment by mat007** [comment]
Thanks for the report, I can confirm this is a bug which I can reproduce.
I have created an internal ticket to have it fixed.
Sorry for the inconvenience!

**Comment by MythreyaK** [comment]
Similar: #4725

**Comment by philippe-granet** [comment]
@mat007 Could you update known issues in the release notes? https://docs.docker.com/docker-for-windows/release-notes/#known-issues

**Comment by mat007** [comment]
@philippe-granet yes good point, we’ll add it to the release notes. Thanks!

**Comment by mat007** [comment]
In the meanwhile a workaround to do it from a Powershell, for instance for 128GB:
```powershell
Resize-VHD -Path 'C:\ProgramData\DockerDesktop\vm-data\DockerDesktop.vhdx' -SizeBytes 128gb
```

**Comment by stephen-turner** [comment]
This is now released in Edge 2.2.3.0, so I'm going to close this ticket.
